ansible-ci/tests/_setup.yml

```diff
@@ -5,11 +5,35 @@
       sf-client --simple instance delete $inst
     done
 
+    attempts=0
+    while [ `sf-client --simple instance list | grep -v uuid | wc -l` -gt 0 ]
+    do
+      attempts=$(( $attempts + 1))
+      sleep 10
+      if [ $attempts -gt 10 ]
+      then
+        echo "Failing after instance deletes did not complete"
+        exit 1
+      fi
+    done
+
     for net in `sf-client --simple network list | grep -v uuid | cut -f 1 -d ","`
     do
       sf-client --simple network delete $net
```

Comment:
This whole code section could probably show case the `sf-client namespace clean` command